views/slices/ImageWithCaption.ejs

```diff
@@ -0,0 +1,28 @@
+<% 
+  const imageUrl = slice.primary.image.url;
+  const caption = ctx.prismicH.asHTML(slice.primary.caption);
+%>
+<div class="post-part single container">
+  <% if (slice.slice_label === 'image-full-width') { %>
+  <div class="blog-header single" style="background-image: url(<%-imageUrl%>)">
+    <div class="wrapper">
+      <% if(caption !== "") { %>
+      <h1>(<%- caption %>)</h1>
+      <% } %>
+    </div>
+  </div>
+  <%} else if (slice.slice_label === 'emphasized') { %>
+  <div class="post-part single container">
+    <p class="block-img emphasized">
+      <img src="<%= imageUrl %>" />
```

Comment:
Why do you sometimes use the HTML escape output (`<%=`) and mostly use the unescaped output (`<%-`)? Is there any reason you use one instead of the other?

Reply:
There were at multiple places where I used `(<%-)` instead of `(<%=)` because I wanted to escape HTML. But after that, to maintain synonymity across the project, I used `(<%-)` only. I have changed here HTML escape output (<%=) to the unescaped output (<%-) too.